USM.Test/AlgorithmTest.cs

```diff
@@ -0,0 +1,509 @@
+﻿using System.Collections.Generic;
+using Xunit;
+using ucsdscheduleme.Repo;
+using ucsdscheduleme.Models;
+using System.Diagnostics;
+using System.Linq;
+
+namespace USM.Test
+{
+    public class TestsAlgorithm
+    {
+        public Course[] Classes()
+        {
+            ScheduleRepo sch = new ScheduleRepo();
+
+            Meeting CSE20Lecture = new Meeting()
```

Comment:
Name these not based on some fake class, but what use this test class provides, (IE ClassWithTwoSections)